src/core/reducers/categories.js

```diff
@@ -51,6 +52,16 @@ export default function categories(state = initialState, action) {
         });
       });
 
+      // Android doesn't have any theme categories but because all lightweight
+      // themes (personas) are installable on Firefox Desktop and Android
+      // we share categories and themes across clientApps.
+      // See: https://github.com/mozilla/addons-frontend/issues/2170
+      //
+      // This can be removed once
+      // https://github.com/mozilla/addons-server/issues/4766 is fixed.
+      categoryList.android[ADDON_TYPE_THEME] = categoryList
```

Comment:
I think it would be nice to log a warning right above this since it would be confusing if we fix the API result and forget to remove this line of code.